nomenclature/codelist.py

```diff
@@ -153,51 +150,51 @@ def from_directory(
 
         """
         code_list, tag_dict = [], CodeList(name="tag")
-
         # parse all files in path if file is None
         file = file or "**/*"
 
-        # parse all files
-        for f in path.glob(f"{file}{ext}"):
-            with open(f, "r", encoding="utf-8") as stream:
-                _code_list = yaml.safe_load(stream)
-
-            # check if this file contains a dictionary with {tag}-style keys
-            if f.name.startswith("tag_"):
-                # validate against the tag schema
-                validate(_code_list, SCHEMA_MAPPING["tag"])
-
-                # cast _codes to `Tag`
-                for item in _code_list:
-                    tag = Tag.from_dict(mapping=item)
-                    tag_dict[tag.name] = [Code.from_dict(a) for a in tag.attributes]
-
-            # if the file does not start with tag, process normally
-            else:
-                # validate the schema of this codelist domain (default `generic`)
-                validate(
-                    _code_list, SCHEMA_MAPPING.get(name, SCHEMA_MAPPING["generic"])
-                )
+        for f in path.glob(file):
+            if f.suffix in {".yaml", ".yml"}:
```

Comment:
Suggest to use this more elaborate formulation to avoid double-indenting
```suggestion
        for file in [f for f in path.glob("**/*") if f.suffix in {".yaml", ".yml"}]:
```